.coveragerc

```diff
@@ -415,6 +415,7 @@ omit =
     homeassistant/components/media_player/volumio.py
     homeassistant/components/media_player/yamaha.py
     homeassistant/components/media_player/yamaha_musiccast.py
+    homeassistant/components/media_player/ziggo_mediabox_xl.py
```

Comment:
**Question:**
Are there any other Ziggo media box models that would support this same API? I'd rather see `ziggo_mediabox.py`, if we ever think this platform will encompass more devices.

Reply:
I'm not the PR author, but all other Ziggo mediaboxes are radically different and can't be controlled using the Mediabox XL API :)

Reply:
OK, perfect! Thanks for the clarification!